src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -129,19 +129,38 @@
   {:render
    (fn [{:keys [state refs]}]
      [:div {}
+      ;; Begin overlay
```

Comment:
I would prefer to avoid this style of comments. I think the distinction is valuable, so my preference would be to pull this out into a function, e.g.:
```
(defn render-overlay [state refs]
  [:div {:style {:visibility (if-not ...
  ...)
```